Portfolio.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>RESUME PAGE</title>
+    <link rel="stylesheet" type="text/css" href="style2.css">
+</head>
+<body>
+        <h4><a href="file:///C:/Users/L/code/fewd/homework/About%20Page.html">ABOUT ME</a> <em>Portfolio</em></h4>
```

Comment:
Use relative links! If this site is anywhere but on your own computer you cannot navigate to this page!